mantidproject/mantid
# Fix RotateInstrumentComponent doc test for win7 debug

Fixes #15204.

Still compares 6 digits but avoids issues with tiny (eps) values that are different from 0, or -tiny values that are printed as "-0" instead of "0"

**To test:**
- Tests should pass
- Review the change and make sure it is sensible

This is an internal change, so no release note.
I'm not very happy that the test (intended also as example code) gets a bit obfuscated. But I didn't find any other way to fix the test given the way we are checking the outputs
